util/perf_count.cc

```diff
@@ -522,104 +525,106 @@ PerformanceCounters * gPerfCounters(&LocalStartupCounters);
-        "WriteWaitLevel0",
-        "WriteNewMem",
-        "WriteError",
-        "WriteNoWait",
-        "GetMem",
-        "GetImm",
-        "GetVersion",
-        "SearchLevel[0]",
-        "SearchLevel[1]",
-        "SearchLevel[2]",
-        "SearchLevel[3]",
-        "SearchLevel[4]",
-        "SearchLevel[5]",
-        "SearchLevel[6]",
-        "TableCached",
-        "TableOpened",
-        "TableGet",
-        "BGCloseUnmap",
-        "BGCompactImm",
-        "BGNormal",
-        "BGCompactLevel0",
-        "BlockFiltered",
-        "BlockFilterFalse",
-        "BlockCached",
-        "BlockRead",
-        "BlockFilterRead",
-        "BlockValidGet",
-        "Debug[0]",
-        "Debug[1]",
-        "Debug[2]",
-        "Debug[3]",
-        "Debug[4]",
-        "ReadBlockError",
-        "DBIterNew",
-        "DBIterNext",
-        "DBIterPrev",
-        "DBIterSeek",
-        "DBIterSeekFirst",
-        "DBIterSeekLast",
-        "DBIterDelete",
-        "eleveldbDirect",
-        "eleveldbQueued",
-        "eleveldbDequeued",
-        "elevelRefCreate",
-        "elevelRefDelete",
-        "ThrottleGauge",
-        "ThrottleCounter",
-        "ThrottleMicros0",
-        "ThrottleKeys0",
-        "ThrottleBacklog0",
-        "ThrottleCompacts0",
-        "ThrottleMicros1",
-        "ThrottleKeys1",
-        "ThrottleBacklog1",
-        "ThrottleCompacts1",
-        "BGWriteError",
-        "ThrottleWait",
-        "ThreadError",
-        "BGImmDirect",
-        "BGImmQueued",
-        "BGImmDequeued",
-        "BGImmWeighted",
-        "BGUnmapDirect",
-        "BGUnmapQueued",
-        "BGUnmapDequeued",
-        "BGUnmapWeighted",
-        "BGLevel0Direct",
-        "BGLevel0Queued",
-        "BGLevel0Dequeued",
-        "BGLevel0Weighted",
-        "BGCompactDirect",
-        "BGCompactQueued",
-        "BGCompactDequeued",
-        "BGCompactWeighted",
-        "FileCacheInsert",
-        "FileCacheRemove",
-        "BlockCacheInsert",
-        "BlockCacheRemove",
-        "ApiDelete",
-        "BGMove",
-        "BGMoveFail",
-        "ThrottleUnadjusted"
+        {"ROFileOpen", true},
+        {"ROFileClose", true},
+        {"ROFileUnmap", true},
+        {"RWFileOpen", true},
+        {"RWFileClose", true},
+        {"RWFileUnmap", true},
+        {"ApiOpen", true},
+        {"ApiGet", true},
+        {"ApiWrite", true},
+        {"WriteSleep", true},
+        {"WriteWaitImm", false},
+        {"WriteWaitLevel0", false},
+        {"WriteNewMem", true},
+        {"WriteError", false},
+        {"WriteNoWait", true},
+        {"GetMem", true},
+        {"GetImm", true},
+        {"GetVersion", true},
+        {"SearchLevel[0]", true},
+        {"SearchLevel[1]", true},
+        {"SearchLevel[2]", true},
+        {"SearchLevel[3]", true},
+        {"SearchLevel[4]", true},
+        {"SearchLevel[5]", true},
+        {"SearchLevel[6]", true},
+        {"TableCached", true},
+        {"TableOpened", true},
```

Comment:
Yes. Since we need a script that the customer needs to set up anyway and it's not exposed to stats yet, the point is moot. I'm fine with this staying like this and activating it along with all the others when needed.